CMakeLists.txt

```diff
@@ -236,6 +236,23 @@ endif()
 add_subdirectory(src/xml/fox)
 
 #===============================================================================
+# RPATH information
+#===============================================================================
+
+# use, i.e. don't skip the full RPATH for the build tree
+set(CMAKE_SKIP_BUILD_RPATH  FALSE)
+
+# when building, don't use the install RPATH already
+# (but later on when installing)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+
+# add the automatically determined parts of the RPATH
+# which point to directories outside the build tree to the install RPATH
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
Based on the description of the variables, it sounds like it should be sufficient to only use this [last command](https://cmake.org/cmake/help/v3.0/variable/CMAKE_INSTALL_RPATH_USE_LINK_PATH.html). The first two are [defaults](https://cmake.org/Wiki/CMake_RPATH_handling) so they can definitely be removed. CMAKE_INSTALL_RPATH seems like it would only be needed for shared libraries inside the build directory (of which we don't have any). CMAKE_INSTALL_RPATH_USE_LINK_PATH should catch both HFD5 and MPI since they are external shared library dependencies. Do you agree?